cudax/include/cuda/experimental/__async/completion_signatures.cuh

```diff
@@ -284,10 +292,10 @@ inline constexpr bool __is_completion_signatures<completion_signatures<_Sigs...>
 
 template <class _Sndr>
 using __is_non_dependent_detail_ = //
-  __mif<__is_completion_signatures<completion_signatures_of_t<_Sndr>>>;
+  _CUDA_VSTD::enable_if_t<__is_completion_signatures<completion_signatures_of_t<_Sndr>>>;
```

Comment:
I am moving towards an more efficient implementation of `__enable_if_t` so we might want to use that

Reply:
I want `cuda::std::enable_if_t` to use the efficient implementation.